.github/workflows/ci-benchmark.yaml

```diff
@@ -56,3 +58,17 @@ jobs:
       - run: ( echo "cat <<EOF >python-sdk/test-connections.yaml"; cat .github/ci-test-connections.yaml; ) >python-sdk/test-connections.yaml && . python-sdk/test-connections.yaml
       - run: python -c 'import os; print(os.getenv("GOOGLE_APPLICATION_CREDENTIALS_JSON", "").strip())' > ${{ env.GOOGLE_APPLICATION_CREDENTIALS }}
       - run: cd python-sdk/tests/benchmark && make
+
+  Astro-deploy:
+    if: >-
+      github.event_name == 'schedule' && github.event.schedule != '0 0 * * 1'
```

Comment:
Similarly I think with this check, our deploy job may not run at all on Mondays.

Reply:
earlier check condition was not right, please let me know I have updated the check